lib/src/http.dart

```diff
@@ -136,7 +148,7 @@ class _PubHttpClient extends http.BaseClient {
   /// Returns a log-formatted string for the HTTP field or header with the given
   /// [name] and [value].
   String _logField(String name, String value) {
-    if (_censoredFields.contains(name.toLowerCase())) {
+    if (!runningFromTest && _censoredFields.contains(name.toLowerCase())) {
```

Comment:
Well in order to test the implementation, I needed the real token to be displayed in the log output - this is and was the way the output headers occur during testing because of the fake server; otherwise it would say 'secret' because the autohorization header is listed as a secure one - which is a good think.

Reply:
The third party server is responsible to make a correct return of the 401. I tested this and I am returning a 401 with the message 'Token not provided or invalid. Please refer to https://mypubserver.com/tokens for more info'
This message is printed to the user.
Furthermore, the current fake server is working correctly for all tests towards official as third party server. I don't think there is a need for a special fake server in this case. The only reason for the `runningFromTest` is to get the token displayed in the log during test to evaluate the correct one.